### ISSUE TITLE
PhotoShow 3.0 : Coming soon !

### ISSUE THREAD

**Issue by thibaud-rohmer** [thibaud-rohmer/PhotoShow#226]
Hey guys !

So, this is not really an "issue", but, I have (great) news.

Most of you don't know this, but the PhotoShow you're using is actually version 2. Version 1 can still be found as another of my github projects.

Well, I'm excited to inform you that **version 3 is in the works** ! 

This version 3 brings a lot of features, as I've been working like crazy on it for the past few weeks. Here is some info : 
- It is NOT a complete rewriting, although a big part of the code has been edited
- Sh*t goes faster, yo. I focused a lot on optimizing stuff. A lot. 
- I'm still adding features and testing the stuff. But as of now, most of it is running fine.
- There WILL be a new "[photoshow-gallery](http://www.photoshow-gallery.com/)" web page, to present the new version. I expect all of you to go have a look at it, admire it, and share it with the whole world. It'll be amazing. (no idea what it's gonna be yet : haven't started it).
- The upgrade will be as easy as always : just "git pull". No worries here.
- As you may have guessed : _you won't need to edit anything when you upgrade_. PhotoShow will upgrade what's needed (for instance, the thumbnails will change automagically). Wow. Much technology.
- No, I don't know WHEN this new version will be released. I really want it to be soon (basically, in a week or so), but one thing's for certain : it won't release until I'm happy with it. 
- No, I haven't seen the last two episodes of Game of Thrones. I was too busy coding.

Oh, right, one more thing : I'm currently working on a specific feature right now, that requires PHP >= 5.5.0. So, I recommend that you update your PHPs =)
For those who don't have it and can't update, no worries : this feature just won't be activated on your systems. It's as simple as that. Oh, and this also is done automagically.

See ya !

TL;DR : PhotoShow 3.0, soon, enjoy.
